lxd/api.go

```diff
@@ -17,7 +17,7 @@ import (
 
 // swagger:operation GET / server api_get
 //
-// Get the supported API enpoints
+// Get the supported API endpoints
```

Comment:
As this is used in the API docs we'll also need a separate commit after regenerating the API docs using `make update-api`, e.g. https://github.com/lxc/lxd/commit/77221976f405246b318af386baa244d4d87f1737